addon/serializers/active-model-serializer.js

```diff
@@ -1,8 +1,8 @@
 import Serializer from '../serializer';
-import { underscore, pluralize, dasherize, singularize } from '../utils/inflector';
+import { underscore, pluralize, dasherize, singularize, camelize } from '../utils/inflector';
 
 export default Serializer.extend({
-
+  normalizeIds: false,
```

Comment:
What is the behavior today if a relationships is passed to this serializer? If it's completely broken I think we could avoid adding this flag.

Reply:
My opinion is that it is broken and doesn't match what EmberData's ResetSerializer expects and #972 documents that. I don't mind avoiding the flag, I just didn't want to create a BC break unnecessarily.

Reply:
Dug back in to this and realized that the big question for me is how ActiveModel should behave. I have no idea so this needs to be settable somehow. I think it should be merged with the flag and then the flag should be defaulted to true at some future point.